.gitignore

```diff
@@ -398,4 +398,5 @@ FodyWeavers.xsd
 *.sln.iml
 
 # Ignore upload files
-Uploads
\ No newline at end of file
+Uploads
+/Pms.Services/Pms.Api/Configuration/DatabaseConfig.cs
```

Comment:
This is not the correct way. If I made some changes for the DatabaseConfig file of the Project then it will not detect it. 
For setting of local database just update appsettings.json and no need to commit and stage it.